File: app/src/main/java/com/subhrajyoti/wallify/gallery/MainViewHolder.java
package com.subhrajyoti.wallify.gallery;

import androidx.recyclerview.widget.RecyclerView;
import android.view.View;
import android.widget.ImageView;

import com.subhrajyoti.wallify.R;

import butterknife.BindView;
import butterknife.ButterKnife;

class MainViewHolder extends RecyclerView.ViewHolder {

    @BindView(R.id.thumbnail)
    ImageView thumbnail;

    MainViewHolder(View itemView) {
        super(itemView);
        ButterKnife.bind(this, itemView);
    }

}

File: app/src/main/java/com/subhrajyoti/wallify/gallery/MainViewHolder.kt
package com.subhrajyoti.wallify.gallery

import android.widget.ImageView
import androidx.recyclerview.widget.RecyclerView.ViewHolder
import com.subhrajyoti.wallify.databinding.RecyclerItemBinding

internal class MainViewHolder(recyclerItemBinding: RecyclerItemBinding) : ViewHolder(recyclerItemBinding.root) {
    val thumbnail: ImageView = recyclerItemBinding.thumbnail
}